plugins/module_utils/policy.py

```diff
@@ -63,30 +91,24 @@ def _hashable_policy(policy, policy_list):
 
     if isinstance(policy, list):
         for each in policy:
-            tupleified = _hashable_policy(each, [])
-            if isinstance(tupleified, list):
-                tupleified = tuple(tupleified)
+            hashed_policy = _hashable_policy(each, [])
+            tupleified = _tuplify_list(hashed_policy)
             policy_list.append(tupleified)
     elif isinstance(policy, string_types) or isinstance(policy, binary_type):
```

Comment:
You can write this like this: `elif isinstance(policy, (binary_type, string_types)):`


```
>>> a = True
>>> isinstance(a, (int, bool))
True
>>> a = 1
>>> isinstance(a, (int, bool))
True
>>> a = "a string"
>>> isinstance(a, (int, bool))
False
```